pkgs/tools/package-management/disnix/dysnomia/default.nix

```diff
@@ -25,16 +28,17 @@ assert enableMongoDatabase -> (mongodb != null && mongodb-tools != null);
 assert enableInfluxDatabase -> influxdb != null;
 assert enableSupervisordProgram -> supervisor != null;
 assert enableDockerContainer -> docker != null;
+assert enableNginxWebApplication -> nginx != null;
+assert enableS6RCService -> s6-rc != null;
+assert enableXinetdService -> xinetd != null;
```

Comment:
If you remove ``? null`` from all packages we do not need this now duplicated check.